src/pudl/package_data/glue/utility_id_pudl.csv

```diff
@@ -4814,7 +4814,7 @@ utility_id_pudl,utility_id_ferc1,utility_name_ferc1,utility_id_eia,utility_name_
 4824,,,60245,Three Peaks Power LLC
 4825,,,60258,Paulding Wind Farm III LLC
 4826,,,60330,"SR South Loving, LLC"
-4827,,,60352,"Golden Fields Solar I, LLC"
+4316,,,60352,"Golden Fields Solar I, LLC"
```

Comment:
this is a bit of a nit but it looks like this one is also up above in the file. can we just remove it?